install/kubernetes/microcks/values.yaml

```diff
@@ -88,7 +94,7 @@ keycloak:
       memory: 400Mi
     limits:
       #cpu: 500m
-      #memory: 512Mi
+      memory: 400Mi
```

Comment:
`512Mi` was inherited from previous benchmarks with older versions of Keycloak. I don't have new benchmarks with KeycloakX that (typically) would make this value lower... Can you please stick with the previous value? That should not impact you as you're not deploying Keycloak and thus it'll not impact existing users as well.

We have to perform new benchmarks in the future but are waiting for some work done regarding Open Telemetry support addition and Grafana dashboard availability.